imp.py

```
@@ -0,0 +1,113 @@
+
+
+def release_lock() -> None:
+    """Release the global import lock."""
+
+    _LOCK.release()
+
+
+def lock_held() -> bool:
+    """Return ``True`` if the import lock is currently held."""
+
+    return _LOCK.locked()
```

Pylint (4.0.5)

[error] Instance of '_RLock' has no 'locked' member

(E1101)